ml-proto/test/stdio_write.wast

```diff
@@ -0,0 +1,14 @@
+(module 
+  (import $write "stdio" "write" (param i32 i32))
+
+  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
```

Comment:
"write" for binary data, "print" for text?

On Fri, Oct 9, 2015 at 12:34 PM, Dan Gohman <notifications@github.com>
wrote:

> In ml-proto/test/stdio_write.wast
> <https://github.com/WebAssembly/spec/pull/121#discussion_r41669560>:
>
> > @@ -0,0 +1,14 @@
> > +(module
> > +  (import $write "stdio" "write" (param i32 i32))
> > +
> > +  (memory 4096 4096 (segment 0 "hello, world!\0D\0A\00"))
>
> Would you mind making a separate test for binary output then? I saw this
> test as the "hello world" test that would be used as an example of what
> wasm code looks like. I don't want to risk anyone thinking that wasm code
> needs a \r to print hello world.
>
> As you say, a separate test for binary output could test \0 and other
> interesting characters.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WebAssembly/spec/pull/121/files#r41669560>.
>